flutter/lib/src/settings/settings_screen.dart

```diff
@@ -21,7 +21,7 @@ class SettingsView extends StatelessWidget {
         // Glue the SettingsManager to the theme selection DropdownButton.
         child: Watch(
           (_) {
-            final themeMode = get<SettingsManager>().themeMode;
+            final themeMode = get<SettingsManager>().themeModeSignal.value;
```

Comment:
Our preference is to rather fix this by adding the missing getter in the manager:

```dart
ThemeMode get themeMode => themeModeSignal.value;
```